CONTRIBUTING.md

```diff
@@ -121,7 +121,7 @@ api:
     privatestate: privatestate.html
 ```
 
-Finally, open `themes/cypress/languages/en.yml` and add an English translation for that sidebar link. In this example we're adding a command so we'll add the following text:
+Finally, open [`themes/cypress/languages/en.yml`](https://github.com/cypress-io/cypress-documentation/blob/develop/themes/cypress/languages/en.yml) and add an English translation for that sidebar link's title. In this example we're adding a command so we'll add the following text:
```

Comment:
I will link the heck out of future submissions; thanks for the steer @jennifer-shehane 👍